engine/apps/api/views/escalation_policy.py

```diff
@@ -21,15 +21,18 @@
 
 class EscalationPolicyView(PublicPrimaryKeyMixin, CreateSerializerMixin, UpdateSerializerMixin, ModelViewSet):
     authentication_classes = (PluginAuthentication,)
-    permission_classes = (IsAuthenticated, ActionPermission)
-    action_permissions = {
-        IsAdmin: (*MODIFY_ACTIONS, "move_to_position"),
-        AnyRole: (
-            *READ_ACTIONS,
-            "escalation_options",
-            "delay_options",
-            "num_minutes_in_window_options",
-        ),
+    permission_classes = (IsAuthenticated, RBACPermission)
+    rbac_permissions = {
+        # TODO: what permissions should go here?
+        "list": [],
```

Comment:
are Escalation "Policies" similar to Escalation "Chains" when it comes to authorization? I am thinking of just using `RBACPermission.Permissions.ESCALATION_CHAINS_<READ|WRITE>` here

Reply:
should be part of `Escalation Chains` permissions